diff -u -U 5 -r --new-file -x *~ -X /tmp/diffexclude jamin-0.9.0/src/scenes.c jamin-0.9.0.doj/src/scenes.c
--- jamin-0.9.0/src/scenes.c	Sun May  9 18:38:40 2004
+++ jamin-0.9.0.doj/src/scenes.c	Thu Oct  7 13:20:51 2004
@@ -58,13 +58,10 @@
 static s_state           scene_state[NUM_SCENES];
 static GdkPixbuf         *LED_green_on = NULL, *LED_green_off = NULL, 
                          *LED_yellow = NULL, *LED_red = NULL;
 
 
-void set_EQ_curve_values ();
-
-
 /*  Initialize all scene related structures and get the widget addresses.  */
 
 void bind_scenes ()
 {
     int             i, j;
@@ -208,11 +205,11 @@
                         /****************************************************/
 
 
                         s_crossfade_to_state (&scene_state[i], -1.0f);
 
-                        set_EQ_curve_values ();
+                        set_EQ_curve_values (0,0.0);
 
                         s_history_add_state (scene_state[i]);
 
                         gtk_image_set_from_pixbuf (l_scene[i], LED_green_on);
                       }
